tests/CMakeLists.txt

```diff
@@ -439,7 +439,7 @@ endfunction()
 function(add_failing_test TESTCMD_BASENAME)
   cmake_parse_arguments(TESTCMD "" "RETVAL;EXE;SCRIPT;SUFFIX" "FILES;ARGS" ${ARGN})
 
-  if ("${TESTCMD_SUFFIX}" STREQUAL "") # Suffix "off" counts as a falsy value, so check diectly for empty string.
```

Comment:
I think the "[falsy](https://en.wiktionary.org/wiki/falsy)" is not actually a typo, but I guess we can keep "false" as it's clear in this context.